### DISCUSSION TITLE
How can I assign an initial value to the output of my network?

### DISCUSSION THREAD

**Discussion by thevelvetunderground** [lululxvi/deepxde#897] [Q&A]
My neural network has two outputs, one of which I need to assign an initial value to, how can I do this?

**Comment by thevelvetunderground** [comment]
And my problem is Time-independent PDE

**Comment by lululxvi** [comment]
Why time-independent PDE has initial value?

**Reply by USSENF6** [reply]
Professor Lu:
I am facing the same issue as this discussion mentioned:
The problem is a biharmonic elasticity problem that puts the deflection _w_ as an output. Normally _w=0_ throughout the domain initially, but deficiencies can bring an initial deflection _w=w*_ to the domain and hence influence the PDE. How can I implement the initial deflection in DEEPXDE?

**Reply by lululxvi** [reply]
Use pre-training/transfer learning
